index.js

```diff
@@ -20,11 +27,20 @@ module.exports = function (req) {
       var opts = options || {};
       opts.expires = new Date(0);
 
-      return !!(req.res.cookie(name, '', opts));
+      try {
+          if (req) {
+              req.res.cookie(name, '', opts)
```

Comment:
if req.res.cookie is undefined, but in that case the delete didn't work and would return false. i could move it outside of the try/catch so it will throw a reasonable thing